lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,119 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
 
-describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+fdescribe('NxPageHeader', function() {
+
+  const renderEl = rtlRenderElement(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+ it('renders a simple NxPageHeader', function() {
+    renderEl();
+    const logo = screen.getByRole('img')
+    expect(logo).toBeInTheDocument();
+    
   });
 
-  it('passes its links, homeLink, and children props to the AbstractNxPageHeader', function() {
-    const props = {
-          links: [{ name: 'foo', href: '#bar' }],
-          homeLink: '#home',
-          children: <button>Click Here For a Free iPhone</button>
-        },
-        component = getShallowComponent(props);
+  it('renders a custom logo when given logo prop', function() {
+    const logo = renderEl({ logo: { path: 'foo', alt: 'bar' } })?.querySelector("img");
+
+    expect(logo).toHaveAttribute('src', 'foo');
+    expect(renderEl({ logo: { path: 'foo', alt: 'bar' } })?.querySelectorAll("img").length).toEqual(1);
+  });
+
+  it('renders logo link when given homeLink prop', function() {
+    renderEl({ homeLink: '#home'});    
 
-    expect(component).toHaveProp('links', props.links);
-    expect(component).toHaveProp('homeLink', props.homeLink);
-    expect(component.children()).toMatchElement(props.children);
+    const homeLink = screen.getByRole('link', {name: 'Home'}),
+      homeLinkLogo = homeLink.querySelector('img');
+
+    expect(homeLink).toBeInTheDocument();
+    expect(homeLinkLogo).toBeInTheDocument();
   });
 
-  it('passes a logo with a webpack-generated path and proper alt tag', function() {
-    const logo = shallow(getShallowComponent().prop('logo'));
+  it('renders links in header when given links prop', function() {
+    renderEl({links: [{ name: 'foo', href: '#bar' }, { name: 'baz', href: '#qux' }]});
+
+    const link1 = screen.getByRole('link', {name: 'foo'}),
+      link2 = screen.getByRole('link', {name: 'baz'})
 
-    // this path comes from imgMock.ts
-    expect(logo).toHaveDisplayName('img');
-    expect(logo).toHaveProp('src', 'path/to/asset.png');
-    expect(logo).toHaveProp('alt', 'Sonatype');
-    expect(logo).toHaveClassName('nx-product__logo-image');
+    expect(link1).toBeInTheDocument();
+    expect(link2).toBeInTheDocument();
   });
```

Comment:
We no longer want to test the `nx-page-header__link--current` class, but we do still want to test that the current link is identified.  The most appropriate way to do that would be to check for the `aria-current` attribute.  The implementation does not currently add that attribute, but improving it such that it does is something that I'd consider to be in-scope for this ticket.

Reply:
> We no longer want to test the `nx-page-header__link--current` class, but we do still want to test that the current link is identified. The most appropriate way to do that would be to check for the [`aria-current`](https://www.w3.org/TR/wai-aria-1.2/#aria-current) attribute (which should be set to `page` on the link to the current page). The implementation does not currently add that attribute, but improving it such that it does is something that I'd consider to be in-scope for this ticket.

Fixed: [793a159](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/793a159d33a6f31bd28f62d24ca85e1cc040d8b1)